services/web/server/src/simcore_service_webserver/storage/_handlers.py

```diff
@@ -109,7 +109,7 @@ def extract_link(data: dict | None) -> str:
     if data is None or "link" not in data:
         raise web.HTTPException(reason=f"No url found in response: '{data}'")
 
-    return data["link"]
+    return parse_obj_as(str, data["link"])
```

Comment:
You could also do this, but I guess you know
```suggestion
    return f"{data['link']}"
```